src/Routing.js

```diff
@@ -121,7 +128,8 @@ export default function Routing ({ appProps }) {
                     enableAdminNavbar={
                       userIsOfficer &&
                       path !== '/2DPrinting' &&
-                      path !== '/3DPrintingForm'
+                      path !== '/3DPrintingForm' &&
```

Comment:
befor the `enableAdminNavbar` prop becomes super long with special routes, lets add an optional field to `signedInRoutes`

```js
// ...
{
  Component: Login,
  path: '/login',
  allowedIf: !userIsAuthenticated,
  redirect: '/',
  // add the prop below
  disableAdminNavbar: true
}
```

then, the prop for `NavbarWrapper` can become
```jsx
enableAdminNavbar={!disableAdminNavbar}
```

Reply:
You would also need to change the `array.map` parameters above, adding `disableAdminNavbar` to it

Reply:
this should be a different PR

Reply:
like navbar-refactor?